certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificatemodel/model/AlertType.java

```diff
@@ -0,0 +1,5 @@
+package se.inera.intyg.certificateservice.domain.certificatemodel.model;
+
+public enum AlertType {
+  INFO, OBSERVE, ERROR
```

Comment:
There is something similar already in the model that is called ElementMessageLevel. Could we do something in order to use the same concept? At least within the model?